examples/extensions/commands/cmd_clean.py

```diff
@@ -0,0 +1,53 @@
+from conan.api.conan_api import ConanAPIV2
+from conans.cli.command import conan_command, OnceArgument
+from conans.cli.output import Color, ConanOutput
+from conans.client.userio import UserInput
+
+info_color = Color.BRIGHT_WHITE
+recipe_color = Color.BRIGHT_BLUE
+removed_color = Color.BRIGHT_YELLOW
+
+
+@conan_command(group="Custom commands")
+def clean(conan_api: ConanAPIV2, parser, *args):
+    """
+    Clean the earlier recipe revisions (and their packages) and the package ones from local or remote.
+
+    Important! Keeps the LATEST package revisions belonging to the LATEST recipe revision.
+    """
```

Comment:
In my opinion the description is a bit confusing... maybe something shorter?
```suggestion
    """
        Deletes (from local cache or remotes) all recipe and package revisions but the latest package revision from the latest recipe revision.
    """
```

Reply:
Much better! Thanks! 👍